docs/index.md

```diff
@@ -1105,22 +1164,43 @@ You need to generate an API key in the softlayer control panel.
 
 Options:
 
-  - `--softlayer-api-endpoint`: Change softlayer API endpoint
-  - `--softlayer-user`: **required** username for your softlayer account, api key needs to match this user.
-  - `--softlayer-api-key`: **required** API key for your user account
+  - `--softlayer-memory`: Memory for host in MB.
+  - `--softlayer-disk-size`: Size of the disk in MB. `0` Sets the softlayer default.
+  - `--softlayer-user`: **required** Username for your softlayer account, api key needs to match this user.
+  - `--softlayer-api-key`: **required** API key for your user account.
+  - `--softlayer-region`: Softlayer region.
   - `--softlayer-cpu`: Number of CPU's for the machine.
-  - `--softlayer-disk-size: Size of the disk in MB. `0` sets the softlayer default.
-  - `--softlayer-domain`: **required** domain name for the machine
-  - `--softlayer-hostname`: hostname for the machine
-  - `--softlayer-hourly-billing`: Sets the hourly billing flag (default), otherwise uses monthly billing
-  - `--softlayer-image`: OS Image to use
+  - `--softlayer-hostname`: Hostname for the machine.
+  - `--softlayer-domain`: **required** Domain name for the machine.
+  - `--softlayer-api-endpoint`: Change softlayer API endpoint.
+  - `--softlayer-hourly-billing`: Sets the hourly billing flag (default), otherwise uses monthly billing.
```

Comment:
"Specifies that hourly billing should be used (default), otherwise monthly billing is used."